redirects.csv

```diff
@@ -74,7 +74,22 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /series/* autoPlay=:autoPlay,/media/series/:splat?autoPlay=:autoPlay,302!
 /series/* sound=:sound,/media/series/:splat?sound=:sound,302!
 /series/*,/media/series/:splat,302!
-/articles/*,/media/articles/:splat,302!
+/media/articles/filters//page/10,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/10,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/2,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/4,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/5,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/6,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/7,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/8,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/9,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/12,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/13,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/14,301!
+/media/articles/filters//page/2,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/15,301!
+/media/articles/filters/+,https://${env:CRDS_MEDIA_ENDPOINT}/articles/,301!
```

Comment:
Similar to the above comment, the organization wants us to account for these poorly constructed hrefs and redirect as appropriate.